plugins/discovery-ec2/build.gradle

```diff
@@ -72,7 +72,10 @@ tasks.register("writeTestJavaPolicy") {
           "permission org.bouncycastle.crypto.CryptoServicesPermission \"exportSecretKey\";",
           "permission org.bouncycastle.crypto.CryptoServicesPermission \"exportPrivateKey\";",
           "permission java.io.FilePermission \"\${javax.net.ssl.trustStore}\", \"read\";",
-          "  permission java.util.PropertyPermission \"com.amazonaws.sdk.ec2MetadataServiceEndpointOverride\", \"write\";",
+          "permission java.util.PropertyPermission \"com.amazonaws.sdk.ec2MetadataServiceEndpointOverride\", \"write\";",
+          "permission java.security.SecurityPermission \"getProperty.jdk.tls.disabledAlgorithms\";",
+          "permission java.security.SecurityPermission \"getProperty.jdk.certpath.disabledAlgorithms\";",
+          "permission java.security.SecurityPermission \"getProperty.keystore.type.compat\";",
```

Comment:
Can you please comment that these permissions are required when running in a JVM that's configured with the BC crypto provider, version x? (assuming I understand the reasoning correctly)

Reply:
I think that is implied by `if (BuildParams.inFipsJvm) {` above. It is true these are new with the new version, but I think git blame can serve as documentation for when these were added to the list. Most probably don't care about the exact permission, this is only used by tests, and there is actually a bigger issue here w.r.t. FIPS and how we identify/notify the user that a plugin when run under FIPS may require additional permissions. In this case (which is the only case), we are fine since these permissions overlap with the required server permissions.

Reply:
Okay, I missed this is only needed in tests.